<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CookieManager</class>
 <widget class="QWidget" name="CookieManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cookies</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/qupzilla.png</normaloff>:/icons/qupzilla.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEdit" name="search"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>These cookies are stored on your computer:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTreeWidget" name="cookieTree">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>22</number>
     </attribute>
     <column>
      <property name="text">
       <string>Server</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cookie name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Server:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Secure:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Expiration:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="SqueezeLabel" name="name">
        <property name="text">
         <string>&lt;cookie not selected&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SqueezeLabel" name="value">
        <property name="text">
         <string>&lt;cookie not selected&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SqueezeLabel" name="server">
        <property name="text">
         <string>&lt;cookie not selected&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="SqueezeLabel" name="path">
        <property name="text">
         <string>&lt;cookie not selected&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="SqueezeLabel" name="secure">
        <property name="text">
         <string>&lt;cookie not selected&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="SqueezeLabel" name="expiration">
        <property name="text">
         <string>&lt;cookie not selected&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="removeAll">
        <property name="text">
         <string>Remove all cookies</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeCookie">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove cookie</string>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SqueezeLabel</class>
   <extends>QLabel</extends>
   <header>squeezelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
